src/components/HomeBody.tsx

```diff
@@ -6,68 +6,109 @@ import { type RootState } from "../contexts/store/store";
 import { getGithubIssueInfo,getGithubIssueList } from "../services/githubApi";
 import Pagination from "./Pagination";
 import styles from "../styles/scss/home-body.module.scss";
+import {Dispatch, UnknownAction} from "redux";
+
+interface ApiParameterState {
+  org : string;
+  repo : string;
+  page : number;
+}
+
+interface IssueProfileUser {
+  user: {
+    avatar_url: string,
+    login: string
+  }
+}
+
+interface IssueContent {
+  content : {
+    number : number,
+    title : string,
+    comments : string,
+    content : string,
+    body : string,
+    labels :[]
+  }
+}
 
 export default function HomeBody() {
 
-  const dispatch = useDispatch();
+  const dispatch :Dispatch<UnknownAction> = useDispatch();
 
   const [openIssuesCount, setOpenIssuesCount] = useState(0);
 
-  const apiParameter = useSelector((state :RootState) => state.apiParameter)
+  const selectedApiParameter : ApiParameterState = useSelector((state :RootState) => state.apiParameter)
+  const selectedOrg : string = selectedApiParameter.org;
+  const selectedRepo : string = selectedApiParameter.repo;
+  const selectedPage : number = selectedApiParameter.page;
```

Comment:
[nits]
```suggestion
  const {org, repo, page} : ApiParameterState = useSelector((state :RootState) => state.apiParameter)
```

上記のように省略記法を使ってもいいかもです。
行数が減って、selectedParameterから何を使うのか（逆に何は使わないのか）が明確になるのでいいです。